Integrations/integration-AlphaSOC_Network_Behavior_Analytics.yml

```diff
@@ -0,0 +1,287 @@
+
+            var threat = threats[threatId];
+            if (!threat) {
+                logInfo('Error: Invalid alert content. Definition for threat "' + threatId + '" not found');
+                continue;
+            }
+
+            if (inScope(threat.severity, threat.policy)) {
+                alert.policy = threat.policy === true ? true : false;
+
+                incidents.push({
+                    "name": threat.title,
+                    "occurred": occurredTs,
+                    "severity": convertSeverity(threat.severity),
+                    "labels": getLabels(alert),
+                    "rawJSON": JSON.stringify(alert)
+                });
+            }
+        }
+    }
+
+    var getOccurredTs = function (alert) {
+        try {
+            return alert.event.ts;
+        } catch (exc) {
+            return null;
+        }
+    }
+
+    var getLabels = function (alert) {
+        var labels = [];
+        if (!alert) {
+            return labels;
+        }
+
+        if (alert.eventType) {
+            labels.push(createLabelEntry('eventType', alert.eventType));
+        }
+
+        if (alert.policy !== undefined && alert.policy !== null) {
+            labels.push(createLabelEntry('policy', alert.policy));
+        }
+
+        if (alert.event) {
+            var eventKeys = Object.keys(alert.event);
+            for (var i = 0; i < eventKeys.length; i++) {
+                var eventKey = eventKeys[i];
+                if (eventKey !== "ts") {
+                    labels.push(createLabelEntry(eventKey, alert.event[eventKey]));
+                }
+            }
+        }
+
+        if (alert.wisdom) {
+            var wisdomKeys = Object.keys(alert.wisdom);
+            for (var i = 0; i < wisdomKeys.length; i++) {
+                var wisdomKey = wisdomKeys[i];
+                labels.push(createLabelEntry('wisdom.' + wisdomKey, alert.wisdom[wisdomKey]));
+            }
+        }
+
+        return labels;
+    }
+
+    var createLabelEntry = function (key, value) {
+        return { 'type': key, 'value': String(value) };
+    }
+
+    var inScope = function (severity, policy) {
+        var severityThereshold = getSeverityThereshold();
+        if (severity === null || severity === undefined || severity < severityThereshold) {
+            return false;
+        }
+
+        var includePolicy = params.policy;
+        if (policy === true && !includePolicy) {
+            return false;
+        }
+
+        return true;
+    }
+
+    var convertSeverity = function (severity) {
+        var demistoSeverity = 0;
+
+        try {
+            demistoSeverity = severity === 1 ? .5 : severity - 1;
+        } catch (exc) {
+            demistoSeverity = 0;
+        }
+
+        return demistoSeverity < 0 || demistoSeverity > 4 ? 0 : demistoSeverity;
+    }
+
+    var getSeverityThereshold = function () {
+        try {
+            var severity = parseInt(params.severity);
```

Comment:
Please handle parameters as constants at the beginning of the code the following.